start-site/src/main/java/io/spring/start/site/extension/dependency/springshell/SpringShellTestBuildCustomizer.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2012-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.start.site.extension.dependency.springshell;
+
+import io.spring.initializr.generator.buildsystem.Build;
+import io.spring.initializr.generator.buildsystem.Dependency;
+import io.spring.initializr.generator.buildsystem.DependencyScope;
+import io.spring.initializr.generator.spring.build.BuildCustomizer;
+
+/**
+ * A {@link BuildCustomizer} that automatically adds {@code spring-shell-test} when Spring
+ * Shell is selected.
+ *
+ * @author DaShaun Carter
+ */
+public class SpringShellTestBuildCustomizer implements BuildCustomizer<Build> {
+
+	@Override
+	public void customize(Build build) {
+		var springShell = Dependency.withCoordinates("org.springframework.shell", "spring-shell-starter-test")
```

Comment:
We don't use `var` in this codebase.

A single statement, as in `SpringSecurityTestBuildCustomizer`, could be used here.